example_unpack_test.go

```diff
@@ -23,7 +23,8 @@ func ExampleUnpackedError_ToJSON_external() {
 	format := eris.NewDefaultFormat(false) // false: omit stack trace
 	u, _ := json.Marshal(uerr.ToJSON(format))
 	fmt.Println(string(u))
-	// Output:
+
+	// example output:
```

Comment:
since we already test these cases elsewhere, i decided it's probably better to make our examples more uniform